t/cli/test_main.sh

```diff
@@ -310,6 +310,35 @@ fi
 
 echo "pass: support environment variables in local_conf"
 
+# support default value when environment not set
+echo '
+etcd:
+    host:
+        - "http://${{ETCD_HOST:192.168.1.1}}:${{ETCD_PORT}}"
```

Comment:
Can we use an other name? I'm worried that ETCD_HOST may have already been set with variables.

Reply:
But ETCD_HOST is used in previous tests, so I reuse this variable

Reply:
OK, let follow this.